nrf_modem/include/nrf_gai_errors.h

```diff
@@ -1,46 +0,0 @@
-/*
```

Comment:
I hope this will be reintroduced in the future?

Reply:
Yes, that's the plan